### PR TITLE
Output regression.diffs in icw_gporca_conan_ubuntu16 task

### PR DESCRIPTION
**Pull request by fanfuxiaoran**
Signed-off-by: Xiaoran Wang <xiwang@pivotal.io>

### CODE REVIEW

**Comment by hlinnaka** [comment]
Note that `make installcheck-world` can leave regression.diffs files in many different directories, not only in `src/test/regress`, if e.g. one of the contrib module's tests fail.

In some of the other scripts, there's a shell trap function called `look4diffs` that searches for regression.diffs and prints it out. Can we use the same mechanism here? I'm not sure how good that `look4diffs` mechanism is, but seems good to be consistent.

File: concourse/scripts/test_gpdb.py [hunk 1]
```diff
@@ -26,6 +27,9 @@ def create_gpadmin_user():
 
 
 def copy_output():
+    diff_files = check_output("find gpdb_src/ -name regression.diffs",shell=True).splitlines()
+    for diff_file in diff_files:
+        subprocess.call("cat "+ diff_file, shell=True)
     shutil.copyfile("gpdb_src/src/test/regress/regression.diffs", "icg_output/regression.diffs")
     shutil.copyfile("gpdb_src/src/test/regress/regression.out", "icg_output/regression.out")
```
Comment (adam8157) [inline concourse/scripts/test_gpdb.py]: Thses copyfile lines should be changed also, I run into other failures than `src/test/regress/regression.diffs` today.

Like:
    subprocess.call("cat " + diff_file + ">> icg_output/regression.diffs", shell=True)

Comment (fanfuxiaoran) [reply concourse/scripts/test_gpdb.py]: Do you mean putting all regresssion.diffs files in one file?

Comment (adam8157) [reply concourse/scripts/test_gpdb.py]: or just remove these two lines, no other jobs are using icg_output.

and please echo a header for every regression.diffs outputing.

Comment (fanfuxiaoran) [reply concourse/scripts/test_gpdb.py]: I don't think it is a good idea to put all regression.diffs files together in one file.

**Pull request merged**